Change Checkbox_radio_button example

Konrud opened the pull request:
Changes for `Checkbox_radio_button` example to conform to the rule stated at the top of the page (i.e. _<input type="radio"> elements, when all radio buttons with the same name value in the form are unchecked_) that is we need more than one radio button in order to see the effect. Moreover, minor changes have been made.

Note: regarding the `Checkbox_radio_button` example I'm not sure whether changes I've made are enough or should it also be changed in some external example that is used by `{{EmbedLiveSample('Checkbox_radio_button', 'auto', 50)}}`. Any help would be much appreciated.

Konrud committed:
Change Checkbox_radio_button example

Changes for `Checkbox_radio_button` example to conform to the rule stated at the top of the page (i.e. _<input type="radio"> elements, when all radio buttons with the same name value in the form are unchecked_) that is we need more than one radio button in order to see the effect. Moreover, minor changes have been made.

wbamberg reviewed (changes requested):
Thanks for this PR, @Konrud , and yes, this is all you need to do to update an example like this.

I like these changes but think we could tweak it a bit. When I first saw this it looked like the two radio buttons were a mistaken duplicate. I think it would help to:
* give the other one a different label (even just something like "This radio label also starts out lime.")
* group them visually together and separate from the checkbox, either with CSS or maybe a `<fieldset>`

What do you think?

Konrud committed:
Customize Checkbox & radio button example

`fieldset` is added to distinguish between the `checkbox` and `radio`. Hidden CSS is added to style the example.

Konrud commented:
@wbamberg , liked your idea. 
I've made a few changes including adding a `fieldset` and some styles using CSS. 
I was wondering if it's possible to have both `hidden` and regular CSS on the page, as I don't want to clutter the page with the styling CSS which is not relevant for the reader. Could you please let me know if it's at all possible?

wbamberg commented:
> I was wondering if it's possible to have both `hidden` and regular CSS on the page, as I don't want to clutter the page with the styling CSS which is not relevant for the reader. Could you please let me know if it's at all possible?

Yes! You can have as many code blocks as you like in there, and make some of them hidden using the `hidden` class, like:

    ```css
    .container {
        display: grid;
    }
    ```

    ```css hidden
    .container {
        margin: 1em;
    }
    ```

Konrud commented:
@wbamberg is there anything else you think should be added to the example or it's good to go?

wbamberg commented:
> @wbamberg is there anything else you think should be added to the example or it's good to go?

No, I think it looks great. I was confused because I thought you were intending to add some hidden CSS but now I see you did already :).

wbamberg reviewed (approved):
Thank you @Konrud , I think this is looking great :).
